If sar cannot open a data file, print a message indicating that sysstat
might not be enabled (closes: #511418).

Index: sysstat/sa_common.c
===================================================================
--- sysstat.orig/sa_common.c	2011-01-19 21:24:59.000000000 +0100
+++ sysstat/sa_common.c	2011-01-19 21:25:05.000000000 +0100
@@ -43,6 +43,7 @@
 #define _(string) (string)
 #endif
 
+static char defaultFileUsed = 0; /* Debian: fix for bug#511418 */
 extern struct act_bitmap cpu_bitmap;
 
 /*
@@ -312,6 +313,7 @@
 	snprintf(datafile, MAX_FILE_LEN,
 		 "%s/sa%02d", SA_DIR, rectime->tm_mday);
 	datafile[MAX_FILE_LEN - 1] = '\0';
+	defaultFileUsed = 1; /* Debian: fix for bug#511418 */
 }
 
 /*
@@ -1058,7 +1060,12 @@
 
 	/* Open sa data file */
 	if ((*ifd = open(dfile, O_RDONLY)) < 0) {
+		const int saved_errno=errno; /* Debian: fix for bug#511418 */
 		fprintf(stderr, _("Cannot open %s: %s\n"), dfile, strerror(errno));
+		if (saved_errno == ENOENT && defaultFileUsed) /* Debian: fix for bug#511418 */
+		{
+			fprintf(stderr, "Please check if data collecting is enabled in /etc/default/sysstat\n");
+		}
 		exit(2);
 	}
 
